Makefile

```diff
@@ -38,4 +40,4 @@ clean:
 		rm **/*.tfstate*
 
 test: fmt
-	GOCACHE=off AWS_PROFILE=cztack-ci-1 go test $(TEST)
+	GOCACHE=off AWS_PROFILE=cztack-ci-1 go test -parallel 10 -test.timeout 30m $(TEST)
```

Comment:
```
Build cache requirement
Go 1.11 will be the last release to support setting the environment variable GOCACHE=off to disable the build cache, introduced in Go 1.10. Starting in Go 1.12, the build cache will be required, as a step toward eliminating $GOPATH/pkg. The module and package loading support described above already require that the build cache be enabled. If you have disabled the build cache to avoid problems you encountered, please file an issue to let us know about them.
```

Reply:
for sometime in the future

Reply:
ok thanks